cumulus_etl/loaders/fhir/ndjson_loader.py

```diff
@@ -37,7 +39,16 @@ def __init__(
         self.until = until
         self.resume = resume
 
-    async def load_all(self, resources: list[str]) -> base.LoaderResults:
+    async def detect_resources(self) -> set[str] | None:
+        if self.root.protocol in {"http", "https"}:
+            return None  # we haven't done the export yet!
```

Comment:
nit - with this and the i2b2 tcp thing below, if i was coming into this codebase fresh, a little more context as to what this means might be helpful (i.e. it's we've got the URL of the place to get the data, not the data itself)

Reply:
OK, I added some more context:

```
            # We haven't done the export yet, so there are no files to inspect yet.
            # Returning None means "dunno" (i.e. "just accept whatever you eventually get").
            return None
```